### DISCUSSION TITLE
Add a 'What's new' button on the admin dashboard when an update is available

### DISCUSSION THREAD

**Discussion by JustRed23** [pelican-dev/panel#830] [Ideas]
As a dev myself, I want to know what changed when the panel gets an update, I need to go to this github and look through the releases section to do that now. Would it be a nice idea to add a button that compares the current version to the newest version and makes a github link like [this](https://github.com/pelican-dev/panel/compare/v1.0.0-beta14...v1.0.0-beta16)?

**Comment by Boy132** [comment]
#1744
